src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function Calculator() {
+  const numbers = ['AC', '+/-', '%', 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, '.'];
+
+  const MathOperators = ['÷', 'x', '-', '+', '='];
+
+  const NumberBtns = numbers.map((number) => (
+    <button className="numberbtns" key={number} type="button">{number}</button>
+  ));
+
+  const operatorBtns = MathOperators.map((operator) => (
+    <button className="numberbtns operatorbtns" key={operator} type="button">{operator}</button>
+  ));
+
+  return (
+    <div className="calcy-container">
+      <div className="calcy-box">
+        <div className="displayvalue">0</div>
+        <div className="Number_container">
+          <div className="Number">
+            {NumberBtns}
+          </div>
+          <div className="operators">
+            {operatorBtns}
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- For learning purpose, kindly use at least one child component for the calculator component, as requested [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component),